src/test/scala/nl/knaw/dans/easy/bagindex/components/AddBagToIndexSpec.scala

```diff
@@ -25,13 +25,14 @@ class AddBagToIndexSpec extends BagIndexDatabaseFixture with AddBagToIndex {
 
   def addBaseTest(): UUID = {
     val bagId = UUID.randomUUID()
+    val doi = "10.5072/dans-x6f-kf66"
```

Comment:
This doi is used several times, so make it a private val. Also it is one that is in the database (bag-index.sql)